reports/tests.py

```diff
@@ -131,37 +102,6 @@ def test_report_view_no_organization(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 404)
 
-    def test_report_view_no_forks(self):
```

Comment:
Same as the previous comments, we're not removing the login/logout views because we still need access to reports and logging in the management console (the view that lists the instances).

Reply:
Same as above, unremoving make it fails as it tries to resolve the registration URL. Need help here too.